dlp/deid.py

```diff
@@ -557,7 +627,10 @@ def write_data(data):
 
     args = parser.parse_args()
 
-    if args.content == 'deid_mask':
+    if args.content == 'deid_replace_with_info_type':
+        deidentify_replace_with_info_type(args.project, args.item,
```

Comment:
This is valid formatting but I prefer to avoid hanging indent:

```python

deidentify_replace_with_info_type(
    args.project, args.item, args.info_types)

```